test/spec/struct.wast

```diff
@@ -105,3 +105,29 @@
   )
   "invalid rtt"
 )
+
+(assert_invalid
+  (module
+    (type $vec (struct (field (ref func))))
+    (func $test
+      (drop
+        ;; too many arguments
+        (struct.new_with_rtt $vec (i32.const 1) (i32.const 2) (rtt.canon $vec))
+      )
+    )
+  )
+  "invalid numbe of arguments to struct.new"
```

Comment:
numbe -> number.. but the message seems to be different anyway? The code says "struct.new must have the right number of operands". The same for the below.

Reply:
Yeah, these text fields are not validated, just that there is an error. One issue is that the text usually comes from the spec tests and the binaryen text might differ (maybe not for a good reason?)